content/english/blog/ep-10.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "PyPodcats 10th Episode"
+meta_title: "PyPodcats 10th Episode"
+description: "Celebrating our 10th episode."
+date: 2025-09-15T05:00:00Z
+categories: ["Blog Post",]
+author: "Mojdeh"
+tags: ["python", "community", "women in tech", "women in Python", "women in open source","python underrespresnted groups"]
+draft: false
+image: "/images/EP19-Una-podcast-cov.gif"
+---
+
+We are back and we are happy to celebrate our 10th episode with Una Galyeva as guest. 
+
+
+ ```[text]
+ I think I tried almost all. I was only mentioning that I didn't try PhD yet. Working for myself, working for someone else, starting my own company, growing my own company, selling my own company.
```

Comment:
```suggestion
 I think I tried almost all. I was only mentioning that I didn't try PhD yet.
 Working for myself, working for someone else, starting my own company, growing my own company, selling my own company.
```

Just trying to break it into different lines. Currently it is rendered with a scrolling bar which makes it hard to read.
You may need to break it into even shorter lines, eg less than 80?